<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>About</class>
 <widget class="QWidget" name="About">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="tTitleLabel" name="titleLabel">
     <property name="text">
      <string>About</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-105</y>
        <width>854</width>
        <height>661</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="centralWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="sysinfoFrame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_32">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="distributionPane" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QGridLayout" name="gridLayout_21">
                <property name="leftMargin">
                 <number>9</number>
                </property>
                <property name="topMargin">
                 <number>9</number>
                </property>
                <property name="rightMargin">
                 <number>9</number>
                </property>
                <property name="bottomMargin">
                 <number>9</number>
                </property>
                <property name="spacing">
                 <number>6</number>
                </property>
                <item row="1" column="1">
                 <widget class="QLabel" name="distroName">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string notr="true">Distribution Name</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0" colspan="2">
                 <widget class="QLabel" name="label_6">
                  <property name="font">
                   <font>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>DISTRIBUTION</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="distroWebpage">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string notr="true">http://www.distro.org/</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_85">
                  <property name="text">
                   <string>You're using</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_16">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>1</height>
                </size>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="hardwareSoftwarePane" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QGridLayout" name="gridLayout_22">
                <property name="leftMargin">
                 <number>9</number>
                </property>
                <property name="topMargin">
                 <number>9</number>
                </property>
                <property name="rightMargin">
                 <number>9</number>
                </property>
                <property name="bottomMargin">
                 <number>9</number>
                </property>
                <property name="spacing">
                 <number>6</number>
                </property>
                <item row="2" column="1">
                 <widget class="tStandardLabel" name="processorType">
                  <property name="text">
                   <string notr="true">Intel</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_89">
                  <property name="text">
                   <string>Available Swap</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0" colspan="2">
                 <widget class="QLabel" name="label_7">
                  <property name="font">
                   <font>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>THIS DEVICE</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="label_91">
                  <property name="text">
                   <string>Kernel Version</string>
                  </property>
                 </widget>
                </item>
                <item row="9" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Device Name</string>
                  </property>
                 </widget>
                </item>
                <item row="8" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Window System</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_88">
                  <property name="text">
                   <string>Available Memory</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Device Type</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="tStandardLabel" name="kernelVersion">
                  <property name="text">
                   <string notr="true">4.11</string>
                  </property>
                 </widget>
                </item>
                <item row="10" column="0" colspan="2">
                 <widget class="QPushButton" name="changeHostnameButton">
                  <property name="text">
                   <string>Change Device Name</string>
                  </property>
                  <property name="icon">
                   <iconset theme="edit-rename"/>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QLabel" name="label_90">
                  <property name="text">
                   <string>Qt Version</string>
                  </property>
                 </widget>
                </item>
                <item row="9" column="1">
                 <widget class="tStandardLabel" name="hostnameLabel">
                  <property name="text">
                   <string notr="true">TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="tStandardLabel" name="availableMemory">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string notr="true">8 GiB</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="tStandardLabel" name="deviceTypeLabel">
                  <property name="text">
                   <string notr="true">TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="tStandardLabel" name="availableSwap">
                  <property name="text">
                   <string notr="true">8 GiB</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="1">
                 <widget class="tStandardLabel" name="qtVersion">
                  <property name="text">
                   <string notr="true">5.9.0</string>
                  </property>
                 </widget>
                </item>
                <item row="8" column="1">
                 <widget class="tStandardLabel" name="windowSystemLabel">
                  <property name="text">
                   <string notr="true">TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_107">
                  <property name="text">
                   <string>Processor</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Video</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="tStandardLabel" name="videoLabel">
                  <property name="text">
                   <string notr="true">AMDx</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="Line" name="displayLine_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>1</height>
                </size>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="aboutPane" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_47">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QLabel" name="versionLabel">
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="text">
                <string notr="true">theDesk %1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>Desktop Environment</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="compileDate">
               <property name="text">
                <string notr="true">You compiled theDesk on %1</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_26">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="acknowledgementsButton">
                 <property name="text">
                  <string>Acknowledgements</string>
                 </property>
                 <property name="icon">
                  <iconset theme="help-about"/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="websiteButton">
                 <property name="text">
                  <string>Website</string>
                 </property>
                 <property name="icon">
                  <iconset theme="text-html">
                   <normaloff>../../../../theShell/shell</normaloff>../../../../theShell/shell</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="bugButton">
                 <property name="text">
                  <string>File Bug</string>
                 </property>
                 <property name="icon">
                  <iconset theme="tools-report-bug">
                   <normaloff>../../../../theShell/shell</normaloff>../../../../theShell/shell</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="sourcesButton">
                 <property name="text">
                  <string>Sources</string>
                 </property>
                 <property name="icon">
                  <iconset theme="commit">
                   <normaloff>../../../../theShell/shell</normaloff>../../../../theShell/shell</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="debugLogButton">
                 <property name="text">
                  <string>View Debug Log</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_42">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_11">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>133</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="Line" name="line_36">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>1</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="tStandardLabel" name="copyrightNotice">
            <property name="text">
             <string notr="true">theDesk is Copyright (c) Victor Tran %1. Licensed under the terms of the GNU General Public License, version 3 or later.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="margin">
             <number>9</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tTitleLabel</class>
   <extends>QLabel</extends>
   <header location="global">ttitlelabel.h</header>
   <slots>
    <signal>backButtonClicked()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>tStandardLabel</class>
   <extends>QLabel</extends>
   <header location="global">tstandardlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
